packages/financial-templates-lib/src/logger/PagerDutyTransport.ts

```diff
@@ -45,8 +46,12 @@ export class PagerDutyTransport extends Transport {
   // Note: info must be any because that's what the base class uses.
   async log(info: any, callback: (error?: unknown) => void): Promise<void> {
     try {
-      // If the message has markdown then add it and the bot-identifier field. Else put the whole info object as a string
-      const logMessage = info.mrkdwn ? info.mrkdwn + `\n${info["bot-identifier"]}` : JSON.stringify(info);
+      // If the message has markdown then add it (with removed anchor text from links that is not supported by PD) and
+      // the bot-identifier field. Else put the whole info object as a string.
+      const logMessage =
+        typeof info.mrkdwn === "string"
```

Comment:
Is this just essentially checking for existence? Or is the markdown sometimes something other than string?

Reply:
Yes. Normally it should be string or undefined if mrkdwn was not added by the application. Since the winston transports expect any for the info, but we later want to parse mrkdwn, it might be safer to check its a string